satellite/metainfo/metainfo.go

```diff
@@ -1100,3 +1100,213 @@ func (endpoint *Endpoint) FinishDeleteObject(ctx context.Context, req *pb.Object
 
 	return &pb.ObjectFinishDeleteResponse{}, nil
 }
+
+// BeginSegment begins segment uploading
+func (endpoint *Endpoint) BeginSegment(ctx context.Context, req *pb.SegmentBeginRequest) (resp *pb.SegmentBeginResponse, err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	streamID, err := endpoint.unmarshalSatStreamID(ctx, req.StreamId)
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, err.Error())
+	}
+
+	_, err = endpoint.validateAuth(ctx, macaroon.Action{
+		Op:            macaroon.ActionWrite,
+		Bucket:        streamID.Bucket,
+		EncryptedPath: streamID.EncryptedPath,
+		Time:          time.Now(),
```

Comment:
should we convert to UTC ?

Reply:
I always had that question. And still confused with time.Now() and time.Now().UTC().

Reply:
I don't think we need to convert it to UTC. At this level `time.Time` will handle timezones.